sdks/python/apache_beam/runners/common.py

```diff
@@ -375,7 +379,7 @@ class DoFnInvoker(object):
   represented by a given DoFnSignature."""
 
   def __init__(self,
-               output_processor,  # type: OutputProcessor
+               output_processor,  # type: _OutputProcessor
```

Comment:
Why this change? (And below.)

Reply:
hmmm... well, at the time that I made this change I _think_ it resolved an error, but either I am mistaken or something changed in the module.   I'm rolling this back.